raft/raft_test.go

```diff
@@ -1492,10 +1494,12 @@ func TestReadOnlyOptionLease(t *testing.T) {
 		wri       uint64
 		wctx      []byte
 	}{
-		{b, 10, 11, []byte("ctx1")},
-		{c, 10, 21, []byte("ctx2")},
-		{b, 10, 31, []byte("ctx3")},
-		{c, 10, 41, []byte("ctx4")},
+		{a, 10, 11, []byte("ctx1")},
+		{b, 10, 21, []byte("ctx2")},
+		{c, 10, 31, []byte("ctx3")},
+		{a, 10, 41, []byte("ctx4")},
```

Comment:
There are 2 cases added(started with `{a, `), which send read index requests to the leader raft. They could cover the corresponding code branch of leader handling read index request directly.